climakitae/selectors.py

```diff
@@ -10,16 +10,20 @@
 import pandas as pd
 import datetime as dt
 from .unit_conversions import _get_unit_conversion_options
-from .catalog_utils import _convert_resolution, _convert_timescale, _convert_scenario
-import pkg_resources 
+from .catalog_utils import (
```

Comment:
Other scripts have a single line between our import statements and our from import statements